snakes_cafe.py

```diff
@@ -0,0 +1,65 @@
+
+menu = """**************************************
+**    Welcome to the Snakes Cafe!   **
+**    Please see our menu below.    **
+**
+** To quit at any time, type "quit" **
+**************************************
+
+Appetizers
+----------
+Wings
+Cookies
+Spring Rolls
+
+Entrees
+-------
+Salmon
+Steak
+Meat Tornado
+A Literal Garden
+
+Desserts
+--------
+Ice Cream
+Cake
+Pie
+
+Drinks
+------
+Coffee
+Tea
+Blood of the Innocent
+
+***********************************
+** What would you like to order? **
+***********************************"""
+
+order = {
+    'Wings': 0,
+    'Cookies': 0,
+    'Spring Rolls': 0,
+    'Salmon': 0,
+    'Steak': 0,
+    'Meat Tornado': 0,
+    'A Literal Garden': 0,
+    'Ice Cream': 0,
+    'Cake': 0,
+    'Pie': 0,
+    'Coffee': 0,
+    'Tea': 0,
+    'Blood of the Innocent': 0
+}
+
+print(menu)
+
+order_line = input('> ')
```

Comment:
It is better practice to wrap this code inside a function instead of just leaving it bare like this. This allows you more control over when the code is executed and keeps your global namespace clean.

Reply:
Once you have wrapped this, you should also add a `if __name__ == '__main__':` block that actually calls the function so that the function only executes when the script is run from the command line.